draft-irtf-cfrg-vdaf.md

```diff
@@ -781,44 +785,43 @@ a particular encoding of that quantity as a byte string.
-  cryptographically secure pseudorandom number generator (CSPRNG). The length
-  of `output` MUST be `len`.
+* `gen_rand(len: int) -> bytes` returns a byte array of the requested length
+  (`len`) generated by a cryptographically secure pseudorandom number generator
+  (CSPRNG).
 
-* `byte(int: int) -> bytes` returns the representation of `int` as a byte
-  string. The value of `int` MUST be in `[0,256)`.
+* `byte(x: int) -> bytes` returns the representation of `x` in range
+  `[0, 256)`.
 
 * `concat(parts: list[bytes]) -> bytes` returns the concatenation of the input
-  byte strings, i.e., `parts[0] || ... || parts[len(parts)-1]`.
+  byte strings, i.e., `parts[0] + ... + parts[len(parts)-1]`.
 
-* `front(length: int, vec: list[Any]) -> (list[Any], list[Any])` splits `vec`
-  into two vectors, where the first vector is made up of the first `length`
-  elements of the input. I.e., `(vec[:length], vec[length:])`.
+* `front(len: int, x: list[Any]) -> tuple[list[Any], list[Any]]` splits `x`
+  into two vectors, where the first vector is made up of the first `len`
+  elements of `x`. I.e., `(x[:len], x[len:])`.
 
 * `xor(left: bytes, right: bytes) -> bytes` returns the bitwise XOR of `left`
   and `right`. An exception is raised if the inputs are not the same length.
 
-* `to_be_bytes(val: int, length: int) -> bytes` converts `val` to big-endian
-  bytes; its value MUST be in the range `[0, 2^(8*length))`, i.e., at least `0`
-  but less than `2^(8*length)`. Function `from_be_bytes(encoded: bytes) -> int`
-  computes the inverse.
+* `to_be_bytes(x: int, len: int) -> bytes` converts an integer `x` whose value
+  is in the range `[0, 2^(8*len))` to big-endian bytes. Function
+  `from_be_bytes(encoded: bytes) -> int` computes the inverse.
 
-* `to_le_bytes(val: int, length: int) -> bytes` converts `val` to little-endian
-  bytes; its value MUST be in the range `[0, 2^(8*length))`. Function
+* `to_le_bytes(x: int, len: int) -> bytes` converts an integer `x` whose value
+  is in the range `[0, 2^(8*len))` to little-endian bytes. Function
   `from_le_bytes(encoded: bytes) -> int` computes the inverse.
 
-* `next_power_of_2(n: int) -> int` returns the smallest integer
-  greater than or equal to `n` that is also a power of two.
+* `next_power_of_2(x: int) -> int` returns the smallest integer
+  greater than or equal to `x` that is also a power of two.
 
-* `additive_secret_share(vec: list[F], num_shares: int, field: type[F])
-  -> list[list[F]]` takes a vector of field elements and returns multiple
-  vectors of the same length, such that they all add up to the input vector,
-  and each proper subset of the vectors are indistinguishable from random.
+* `additive_secret_share(x: list[F], num_shares: int, field: type[F]) ->
+  list[list[F]]` takes a vector `x` of field elements and returns multiple
+  vectors of the same length, such that they all add up to the input vector.
+  Note that this function is not used normatively in this document.
```

Comment:
this method is only used for `run_flp()` and we don't care if it's an actually secure secret sharing.